packages/nimble-components/src/rich-text/base/models/rich-text-validator.ts

```diff
@@ -0,0 +1,45 @@
+import type { RichTextMention } from '../../../rich-text-mention/base';
+import type { RichTextValidity } from '../types';
+
+/**
+ * Helper class for the nimble rich text components to validate that the configuration
+ * is valid and report which aspects of the configuration are valid or invalid.
+ */
+export class RichTextValidator {
```

Comment:
For each of the invalid cases we should have a test coverage that ensure `isValid()` is `false` and `getValidity()` returns an object with the right field set. You can look at the table validator spec as an example.

Reply:
Covered the test cases in editor and viewer spec files